src/js/components/modals/AlertModalComponent.jsx

```diff
@@ -12,7 +13,15 @@ var AlertModalComponent = React.createClass({
   },
 
   componentDidMount: function () {
-    React.findDOMNode(this.refs.button).focus();
+    let refs = this.refs;
+
+    React.findDOMNode(refs.button).focus();
+    Mousetrap(React.findDOMNode(refs.modalComponent))
+      .bind("esc", this.handleDestroy);
```

Comment:
This is giving me eslint complaints, eg
`A function with a name starting with an uppercase letter should only be used as a constructor  new-cap`

Can we add eslint ignore directives, or move to the `bind` syntax used in Marathon.jsx?